src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -44,10 +52,10 @@ public Message(byte[] array)
         /// <remarks>
         /// The easiest way to create a new message from a string is the following:
         /// <code>
-        /// message.Body = System.Text.Encoding.UTF8.GetBytes("Message1");
+        /// message.Body = new ArrySegment&lt;byte&gt;(System.Text.Encoding.UTF8.GetBytes("Message1"));
         /// </code>
         /// </remarks>
-        public byte[] Body { get; set; }
+        public ArraySegment<byte> Body { get; set; }
```

Comment:
This feels off. To create a new message with payload, ctor accepts `byte[]`.
To assign a body later I need to use `ArraySegment<byte>`. Why can't it be `byte[]` all the way?